tests/foreman/cli/test_repository.py

```diff
@@ -982,7 +982,45 @@ def test_positive_synchronize_docker_repo_set_tags_later(self, repo):
                     'content-type': 'docker',
                     'docker-upstream-name': CONTAINER_UPSTREAM_NAME,
                     'url': CONTAINER_REGISTRY_HUB,
-                    'docker-tags-whitelist': f"latest,{gen_string('alpha')}",
+                    'mirroring-policy': 'mirror_content_only',
+                }
+            ]
+        ),
+        indirect=True,
+    )
+    def test_positive_synchronize_docker_repo_set_tags_later_content_only(self, repo):
+        """Verify that adding tags whitelist and re-syncing after
+        synchronizing full repository doesn't remove content that was
```

Comment:
Description is same as previous test, I wonder do you mean:
```suggestion
        synchronizing full repository does remove content that was
```

Reply:
yes that was a typo. thanks for catching